unit-tests/src/test/scala/scala/scalanative/posix/TimeTest.scala

```diff
@@ -38,29 +58,28 @@ class TimeTest {
       anno_zero_ptr.tm_wday = 1
       val cstr: CString = asctime_r(anno_zero_ptr, alloc[Byte](26))
       val str: String   = fromCString(cstr)
-      assertTrue("Mon Jan  1 00:00:00 1900\n".equals(str))
+      assertEquals("Mon Jan  1 00:00:00 1900\n", str)
     }
   }
-
   @Test def localtimeShouldTransformTheEpochToLocaltime(): Unit = {
     val time_ptr = stackalloc[time_t]
-    !time_ptr = epoch + timezone
+    !time_ptr = epoch + gmtOffset
```

Comment:
This was a good effort to make the tests portable to any time, but it would fail when clocks are in daylight saving time. According to the [POSIX specification](https://pubs.opengroup.org/onlinepubs/9699919799/):
> The external variable timezone shall be set to the difference, in seconds, between Coordinated Universal Time (UTC) and local standard time